projects/angular/src/emphasis/alert/alert.ts

```diff
@@ -73,13 +76,19 @@ export class ClrAlert implements OnInit, OnDestroy {
     return this.iconService.iconInfoFromType(this.iconService.alertType).cssClass;
   }
 
-  @HostBinding('class.alert-hidden')
   get hidden() {
     return this._hidden;
```

Comment:
Does initializing `this._hidden = false` resolve this issue without the additional logic to manually add/remove the class?

Reply:
The error happens when a change is detected `undefined ` to `false` or `false` to `true` it doesn't matter what the default value is, it will happen with default value `true` to `false` as well.
The error lies in the way Angular makes additional check detection in DEV mode. More here: https://angular.io/errors/NG0100